chat-plugins/info.js

```diff
@@ -338,6 +338,9 @@ exports.commands = {
 		if (sep[1] && toId(sep[1]) in Tools.dexes) {
 			mod = Tools.mod(toId(sep[1]));
 		}
+		else if (sep[1] && Tools.getFormat(sep[1]).mod) {
```

Comment:
I think PS convention for this would be to have this on the same line as the one above it